### DISCUSSION TITLE
Rename all Styles .axaml files

### DISCUSSION THREAD

**Discussion by robloo** [amwx/FluentAvalonia#193] [General]
The current naming convention for control theme resource dictionary files doesn't make sense anymore. "CalendarStyles.axaml" should probably be renamed to just "Calendar.axaml" for example as in other XAML frameworks.

These resource dictionaries now contain mostly control themes (not styles) and should be more generically names. What do you think @amwx?

**Comment by amwx** [comment]
I did briefly think about that when I began the ControlTheme porting. In naming those files originally, it wasn't so much they contained `Styles` (Avalonia definition) but more they were the styles for a given control (MS/WPF/WinUI definition), so I decided to keep what I had. It really doesn't matter to me. If anyone else sees this and voices in favor of changing the name then I'll go ahead and change it.
